docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -1,175 +1,268 @@
-====
-Each data stream requires a matching
-<<create-index-template,index template>>. The stream uses this
-template to create new backing indices.
+To view a list of available snapshots in {kib}, go to the main menu and click
+*Stack Management > Snapshot and Restore*.
 
-When restoring a data stream, ensure a matching template exists for the stream.
-You can do this using one of the following methods:
+You can also use the <<get-snapshot-repo-api,get repository API>> to fetch a
+list of registered snapshot repositories.
 
-* Check for existing templates that match the stream. If no matching template
-  exists, <<create-index-template,create one>>.
+[source,console]
+----
+GET _snapshot
+----
+// TEST[setup:setup-snapshots]
 
-* Restore a global cluster state that includes a matching template for the
-  stream.
+Use the <<get-snapshot-api,get snapshot API>> to retrieve snapshots for a
+repository.
 
-If no index template matches a data stream, the stream cannot
-<<manually-roll-over-a-data-stream,roll over>> or create new backing indices.
-====
+[source,console]
+----
+GET _snapshot/my_repository/*?verbose=false
+----
+// TEST[setup:setup-snapshots]
+
+[discrete]
+[[restore-index-data-stream]]
+=== Restore an index or data stream
 
-The `rename_pattern`
-and `rename_replacement` options can be also used to rename data streams and indices on restore
-using regular expression that supports referencing the original text, according to the https://docs.oracle.com/javase/8/docs/api/java/util/regex/Matcher.html#appendReplacement-java.lang.StringBuffer-java.lang.String-[`appendReplacement`] logic.
+You can restore an index or data stream from a snapshot using {kib}'s *Snapshot
+and Restore* feature or the <<restore-snapshot-api,restore snapshot API>>.
 
-[[rename-restored-data-stream]]
-// tag::rename-restored-data-stream-tag[]
-If you rename a restored data stream, its backing indices are also
-renamed. For example, if you rename the `logs` data stream to `restored-logs`,
-the backing index `.ds-logs-2099.03.09-000005` is renamed to
-`.ds-restored-logs-2099.03.09-000005`.
+You can only restore an existing index if it's closed. You can't restore an
+existing data stream. Use one of the following methods to avoid conflicts with
+existing indices or data streams:
 
-[WARNING]
-====
-If you rename a restored stream, ensure an index template matches the new stream
-name. If no index template matches the stream, it cannot
-<<manually-roll-over-a-data-stream,roll over>> or create new backing indices.
-====
-// end::rename-restored-data-stream-tag[]
+* <<delete-restore>>
+* <<close-restore>>
+* <<rename-on-restore>>
 
-To prevent aliases from being restored with their associated data streams and
-indices, set `include_aliases` to `false`.
+[discrete]
+[[delete-restore]]
+==== Delete and restore
+
+The simplest way to avoid conflicts is to delete an existing index or data
+stream before restoring it. To prevent the accidental recreation of the index or
```

Comment:
`before [restoring](https://www.elastic.co/guide/en/elasticsearch/reference/current/restore-snapshot-api.html) it`

Reply:
I link to the API above. The Kibana feature is also a valid method.